meta/main.yml

```diff
@@ -10,6 +10,10 @@ dependencies:
   - role: debops.golang
     tags: [ 'depend::golang', 'depend::golang:gitlab', 'depend-of::gitlab', 'type::dependency:hard' ]
 
+  - role: debops.nodejs
+    tags: [ 'depend::nodejs', 'depend::nodejs:gitlab', 'depend-of::gitlab', 'type::dependency:hard' ]
+    nodejs__upstream: True
```

Comment:
> So, GitLab now requires NodeJS to work?

[Yes.](https://docs.gitlab.com/ce/install/installation.html#4-node)

> Is a minimum version required or will GitLab work with NodeJS from a given OS release (Jessie, Xenial)?

The documentation says  node >= v4.3.0 is required (jessie has 0.10, xenial 4.2), but for some reason webpack fails to install properly using the packages from https://deb.nodesource.com/node_4.x on jessie.

https://deb.nodesource.com/node_6.x works fine.